.devcontainer/Containerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.access.redhat.com/ubi9/python-311:1-52
+
+USER root
+
+COPY playground/tests/requirements.txt .
+
+RUN pip3 install -r requirements.txt && \
+    dnf install -y podman buildah && \
```

Comment:
podman and buildah don't work without additional modifications in a container. You have to switch to a different storage driver as the overlay driver is not available inside containers

Reply:
Thanks for your input. I'll have to consult with folks who have more knowledge and experience with it.

Reply:
It is probably not the problem. I've invoked the container and the curl command mainauly and it worked as expected.

Reply:
Found the problem. Unrelated to the plugin. Thanks.